### PR TITLE
2448: Fix google api error

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Upgrade fastlane to v2.216.0 and set SUPPLY_UPLOAD_MAX_RETRIES to fix google api issue.

### Proposed changes

- Upgrade fastlane to v2.216.0 in native, android and ios
- Set SUPPLY_UPLOAD_MAX_RETRIES to 5 to automatically retry randomly failing upload to playstore

### Side effects

None.

### Resolved issues

Fixes: #2448.

---

### CODE REVIEW

File: native/fastlane/Fastfile [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+fastlane_version "2.216.0"
 
 require_relative "read_build_config"
 
 require "yaml"
 require "base64"
 
+# Google API fails randomly, try to retry
+# https://github.com/fastlane/fastlane/issues/21507#issuecomment-1723116829
+ENV['SUPPLY_UPLOAD_MAX_RETRIES']='5'
```
Comment (LeandraH) [inline native/fastlane/Fastfile:10]: Did you already open a task to remove this?

**Review by LeandraH — APPROVED** [review]
Awesome, thanks for fixing it!

**Review by ztefanie — APPROVED** [review]
Thanks for fixing this.

**Comment by LeandraH** [comment]
I'll merge since Steffen is off this week and it would be nice to have this before Monday.

**Pull request merged**